lib/google_search_data_viewer_web/controllers/user_controller.ex

```diff
@@ -0,0 +1,26 @@
+defmodule GoogleSearchDataViewerWeb.UserController do
+  use GoogleSearchDataViewerWeb, :controller
+
+  alias GoogleSearchDataViewer.Accounts
+  alias GoogleSearchDataViewer.Accounts.User
+
```

Comment:
Please remove this empty line to make the `alias` consistent.

Reply:
Done, ty.